products.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>Sprint Challenge - About</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+
+    <div class="container">
+
+        <header>
+            <nav class="nav-main">
+                <img src="img/lambda-black.png" alt="Lambda Logo">
+
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="products.html">Products</a>
+                <a href="">Blog</a>
+                <a href="">Contact</a>
+            </nav>
+        </header>
+
+        <img class="jumbo-img" src="https://static1.squarespace.com/static/58b6f8f8d2b85748a09f4c75/t/5aac764b8a922d6d3c9e40a3/1521251925697/Official+RR+Merch+Banner.png?format=1500w"
+            alt="Merchandise Banner">
+
+        <section class="products-about-section">
+            <div class="explosion">
+                <h1>Merchandise</h1>
+            </div>
+        </section>
+
+        <section class="products-card-section">
+            <div class="card">
+                <img src="https://static.wixstatic.com/media/1f6a5f_ded6a749d9f94c63aaa0673a3683cab2~mv2_d_3000_3000_s_4_2.png/v1/fill/w_1120,h_1120,q_85,usm_0.66_1.00_0.01/1f6a5f_ded6a749d9f94c63aaa0673a3683cab2~mv2_d_3000_3000_s_4_2.png"
+                    alt="Hat">
+                <h2>Lambda Shield Hat</h2>
+                <p>All proceeds generated from purchases within the Lambda Swag Shop are invested directly into the Lambda School
+                    Perpetual Fund. Your purchase provides Lambda students with the aid they need to fully pursue their education.</p>
+                <button onclick="location.href='https://www.lambdaschoolshop.com/product-page/lambda-shield-hat'" type="button">Buy</button>
```

Comment:
excellent stretch! one thing to keep in mind is that you can often use an anchor tag in this scenario and style it to look like a button. this means you don't need to use javascript, which some people have disabled, so it's slightly more accessible